### PR TITLE
Added upgrade docs for v13

### PR DESCRIPTION
**Pull request by wisyhambolu**
- Added docs for Altis v13 beta release
- humanmade/product-dev#1225

### CODE REVIEW

File: other-docs/guides/upgrading/v13.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+### PHP 7.4 ###
+
+The release of Altis v13 introduces the removal of support for PHP 7.4, follow the [PHP Version Guide](docs://guides/updating-php/) for information on how to upgrade to PHP 8.0 or PHP 8.1.
+
+
+## Headline Features
+
+### WordPress 6.0.1 ###
+
+Wordpress 6.0.1 brings with it a lot of performance improvements to the block editor, taxonomies and the cache API amongsts many others.
```
Comment (kovshenin) [inline other-docs/guides/upgrading/v13.md:53]: WordPress with a capital P please. Also I think this section should be about WordPress 6.0 as that's the new major release in Altis v13, 6.0.1 and 6.0.2 are bugfixes/maintenance/security patches.

File: other-docs/guides/upgrading/v13.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- [Improved block locking](https://make.wordpress.org/core/2022/05/05/block-locking-settings-in-wordpress-6-0/)
+- [Enhanced template block customization](https://make.wordpress.org/core/2022/05/05/updates-to-the-wordpress-create-block-templating-system/)
+- [Full CRUD options for batch API cache operations](https://make.wordpress.org/core/2022/04/29/caching-improvements-in-wordpress-6-0/)
+- [Storage of file size in Media Metadata](https://make.wordpress.org/core/2022/05/02/media-storing-file-size-as-part-of-metadata/)
+- [Taxonomy performance improvements](https://make.wordpress.org/core/2022/04/28/taxonomy-performance-improvements-in-wordpress-6-0/)
+- [Support for the creation of multiple `theme.json` variations](https://make.wordpress.org/core/2022/05/03/global-styles-variations-in-wordpress-6-0/)
+
+
+## New User Documentation ###
+We have released a first version of user-oriented documentation for Altis.  These are less technical articles, geared more towards letting you know what all the buttons do. There are almost 30 articles to get you started and we will be improving the ones we have and adding more over time.
```
Comment (kovshenin) [inline other-docs/guides/upgrading/v13.md:66]: We should have a link to these new docs here.

**Review by kovshenin — APPROVED** [review]

File: other-docs/guides/upgrading/v13.md (hunk 1 above)
Comment (wisyhambolu) [reply other-docs/guides/upgrading/v13.md]: Thanks for pointing this out, it has been corrected now.

File: other-docs/guides/upgrading/v13.md (hunk 2 above)
Comment (wisyhambolu) [reply other-docs/guides/upgrading/v13.md]: @kovshenin I've added the link for the user-docs.

File: other-docs/guides/upgrading/v13.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- [Enhanced template block customization](https://make.wordpress.org/core/2022/05/05/updates-to-the-wordpress-create-block-templating-system/)
+- [Full CRUD options for batch API cache operations](https://make.wordpress.org/core/2022/04/29/caching-improvements-in-wordpress-6-0/)
+- [Storage of file size in Media Metadata](https://make.wordpress.org/core/2022/05/02/media-storing-file-size-as-part-of-metadata/)
+- [Taxonomy performance improvements](https://make.wordpress.org/core/2022/04/28/taxonomy-performance-improvements-in-wordpress-6-0/)
+- [Support for the creation of multiple `theme.json` variations](https://make.wordpress.org/core/2022/05/03/global-styles-variations-in-wordpress-6-0/)
+
+
+## New User Documentation ###
+We have released a first version of user-oriented documentation for Altis which can be found at https://help.altis-dxp.com/. These are less technical articles, geared more towards letting you know what all the buttons do. There are almost 30 articles to get you started and we will be improving the ones we have and adding more over time. The user documentation will also be available under `Documentation > User Guides` in a site’s Dashboard.
```
Comment (mikelittle) [inline other-docs/guides/upgrading/v13.md:66]: " in a site's Dashboard" > "in your site's Dashboard"

**Review by mikelittle — CHANGES_REQUESTED** [review]
This looks good. Just one minor change.

File: other-docs/guides/upgrading/v13.md (hunk 3 above)
Comment (wisyhambolu) [reply other-docs/guides/upgrading/v13.md]: Oh yeah, thanks. I've corrected this.

**Review by mikelittle — APPROVED** [review]
Thanks for the change.
Looks good to me.

**Pull request merged**